redun/executors/docker.py

```diff
@@ -69,27 +79,28 @@ def run_docker(
         Number of GB of shared memory to reserve for the container.
     """
     # Add AWS credentials to environment for docker command.
-    env = dict(os.environ)
-    if not aws_utils.is_ec2_instance():
-        env.update(aws_utils.get_aws_env_vars())
-
     common_args = []
     if cleanup:
         common_args.append("--rm")
 
-    # Environment args.
-    common_args.extend(
-        [
-            "-e",
-            "AWS_ACCESS_KEY_ID",
-            "-e",
-            "AWS_SECRET_ACCESS_KEY",
-            "-e",
-            "AWS_SESSION_TOKEN",
-            "-e",
-            "AWS_DEFAULT_REGION",
-        ]
-    )
+    env = dict(os.environ)
+    if executor_name == "aws_batch-debug" and not aws_utils.is_ec2_instance():
+        env.update(aws_utils.get_aws_env_vars())
+        # AWS Environment args.
+        common_args.extend(
+            [
+                "-e",
+                "AWS_ACCESS_KEY_ID",
+                "-e",
+                "AWS_SECRET_ACCESS_KEY",
+                "-e",
+                "AWS_SESSION_TOKEN",
+                "-e",
+                "AWS_DEFAULT_REGION",
+            ]
+        )
+    elif executor_name == "gcp_batch_debug":
+        pass
```

Comment:
hmm, if we don't need to forward anything to gcp then we can just remove those two lines?